core_backend/app/llm_call/parse_input.py

```diff
@@ -116,6 +116,48 @@ async def wrapper(
     return wrapper
 
 
+def _process_identified_language_response(
+    identified_language: IdentifiedLanguage | None,
+    response: UserQueryResponse,
+    task: str = "LANGUAGE IDENTIFICATION",
+) -> UserQueryResponse | UserQueryResponseError:
+    """Process the identified language and return the response."""
+    supported_languages = IdentifiedLanguage.get_supported_languages()
+    supported_language_string = ", ".join(supported_languages)
+
+    match identified_language:
+        case IdentifiedLanguage.UNINTELLIGIBLE:
+            error_response = UserQueryResponseError(
+                error_message=STANDARD_FAILURE_MESSAGE
+                + " Unintelligible question. The following languages are supported: "
```

Comment:
input not question - this is also valid for embedding search which doesn't need questions necessarily.

```suggestion
                + " Unintelligible input. The following languages are supported: "
```